tools/docs/datasources.ts

```diff
@@ -39,24 +43,30 @@ export async function generateDatasources(
       # ${displayName} Datasource
       `;
     md += '\n\n';
-    md += `**Identifier**: \`${id}\`\n\n`;
+
+    let tableContent = '## Table of values\n\n';
+
+    tableContent += '| Name | Vaue | Notes |\n';
+    tableContent += '| :-- | :-- | :-- |\n';
+
+    tableContent += `| Identifier | \`${id}\` | \n`;
     if (defaultVersioning) {
-      md += `**Default versioning**: \`${defaultVersioning}\`\n\n`;
+      tableContent += `| Default versioning | \`${defaultVersioning}\` | \n`;
     } else {
-      md += `**Default versioning**: no default versioning\n\n`;
+      tableContent += `| Default versioning | No default versioning | \n`;
     }
+
+    tableContent += `| Custom registry support | ${customRegistrySupport ? 'Yes' : 'No'} | \n`;
```

Comment:
A specific reason we are no longer using the icons?

Also I would put custom registry support as first row, as I would assume this is the thing searched most often by users.

Reply:
Nothing specific about the icons. @HonkingGoose wdyt??

Reply:
I find plain text is a bit easier on screenreaders. But emojis will get "read" as well, like this:

🙂 becomes: "Slightly smiling face emoji", or something similar to that at least.

I'm fine with moving the columns so that the first item is the thing users are most likely to want to know first. 🙂

Reply:
But we should keep the order of the columns logical, so housekeeping columns go first.

Reply:
I like the emojies 🙃